# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditAggregateTransaction.h
# Commit: b9ebe58001d7
# Full Hash: b9ebe58001d787bb0bbad6d39ae8681966d5a77f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r ce2087ddab5c -r b9ebe58001d7 editor/libeditor/EditAggregateTransaction.h
--- a/editor/libeditor/EditAggregateTransaction.h	Fri Apr 03 08:30:37 2020 +0000
+++ b/editor/libeditor/EditAggregateTransaction.h	Fri Apr 03 08:32:01 2020 +0000
@@ -43,7 +43,7 @@
 
   NS_DECL_EDITTRANSACTIONBASE
 
-  NS_IMETHOD RedoTransaction() override;
+  MOZ_CAN_RUN_SCRIPT NS_IMETHOD RedoTransaction() override;
   NS_IMETHOD Merge(nsITransaction* aTransaction, bool* aDidMerge) override;
 
   /**